pkg/netid/netid.go

```diff
@@ -0,0 +1,94 @@
+package netid
+
+/*
+Accessor methods to annotate namespace for multitenant support
+*/
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+)
+
+const (
+	// Maximum VXLAN Virtual Network Identifier(VNID) as per RFC#7348
+	MaxVNID = uint((1 << 24) - 1)
+	// VNID: 1 to 9 are internally reserved for any special cases in the future
+	MinVNID = 10
```

Comment:
is this a new restriction? what happens if there are existing vinds in this range? Do we have a good sense what we'll use the "special" ones for, and is 10 enough?